devito/function.py

```diff
@@ -1192,7 +1162,7 @@ def inject(self, field, expr, offset=0):
 
         # Substitute coordinate base symbols into the coefficients
         return [Inc(field.subs(vsub),
-                    field.subs(vsub) + expr.subs(subs).subs(vsub) * b.subs(subs))
+                    Add(field.subs(vsub) + expr.subs(subs).subs(vsub) * b.subs(subs)))
```

Comment:
the question is: why do we need an ``Add`` here? shouldn't it be a Differentiable already?

Reply:
Leftover from early stuff, shouldn't be needed